tests/Liquid/StandardFiltersTest.php

```diff
@@ -792,7 +792,11 @@ public function testSplit()
 		$data = array(
 			array(
 				'',
-				array(0 => ''),
+				array(),
```

Comment:
Is this a BC break?

Reply:
Personally I lean towards no but I could see the potential argument for yes.